src/test/java/redis/clients/jedis/tests/commands/ClusterBinaryJedisCommandsTest.java

```diff
@@ -166,6 +166,15 @@ public void testRpush() {
   }
 
   @Test
+  public void testKeys() {
+    assertEquals(0, jedisCluster.keys("{f}o*".getBytes()).size());
+    jedisCluster.set("{f}oo1".getBytes(), "bar".getBytes());
+    jedisCluster.set("{f}oo2".getBytes(), "bar".getBytes());
+    jedisCluster.set("{f}oo3".getBytes(), "bar".getBytes());
+    assertEquals(3, jedisCluster.keys("{f}o*".getBytes()).size());
```

Comment:
Let's add the fail case as well via another method or current method with proper try-catch.

Reply:
@HeartSaVioR Please check the new commit.